Set-Video-Mode/README.md

```diff
@@ -0,0 +1,105 @@
+# script-plugin-set-video-mode
+
+This repository contains a example of using Script Plugin and Plugin Message feature to set BrightSign VideoMode.
+
+videomode_plugin.brs is a custom BrightSign plugin example that you can configure videoMode for series 4 players via Plugin Message Usage.
+
+## Plugin Message Usage
+
+1. Update videomode_plugin.brs with your expected screens settings
+2. Add videomode_plugin.brs to BrightAuthor or BrightAuthor:Connected (Presentation Settings > Support Content > Script Plugin)
+3. Add 'videomode' to your Script plugin name. Make sure that the assigned name matches the <plugin_name> specified in your BRs plugin <plugin_name>_Initialize() function. 
```

Comment:
** Capitalize whole word - _BRS_

Reply:
Updated in [730000e](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/730000e99c8289a24d7a7f658c6f0f989cd53661)